DeleteUserDataActivity/models/message_status.ts

```diff
@@ -0,0 +1,41 @@
+import * as DocumentDb from "documentdb";
+import { Either } from "fp-ts/lib/Either";
+import {
+  MESSAGE_STATUS_MODEL_ID_FIELD,
+  MESSAGE_STATUS_MODEL_PK_FIELD,
+  MessageStatusModel as MessageStatusModelBase,
+  RetrievedMessageStatus
+} from "io-functions-commons/dist/src/models/message_status";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
+import * as DocumentDbUtils from "../utils/documentdb";
+
+export class MessageStatusModel extends MessageStatusModelBase {
+  public async deleteMessageStatusVersion(
+    messageId: NonEmptyString,
+    documentId: NonEmptyString
+  ): Promise<Either<DocumentDb.QueryError, string>> {
+    return DocumentDbUtils.deleteDocument(
+      this.dbClient,
+      this.collectionUri,
+      documentId,
+      messageId
+    );
+  }
```

Comment:
can't we just use `DocumentDbUtils.deleteDocument` where needed without extending the models ?